addons/knowledge/models/knowledge_article.py

```diff
@@ -238,6 +241,18 @@ def _compute_inherited_permission(self):
             articles.inherited_permission = ancestors[-1:].internal_permission
             articles.inherited_permission_parent_id = ancestors[-1:]
 
+    def _compute_user_is_admin(self):
+        for article in self:
+            article.user_is_admin = self.env.su or self.env.user.has_group('base.group_system')
+
+    def _search_user_is_admin(self, operator, value):
+        if operator not in ('=', '!=') or not isinstance(value, bool):
+            raise NotImplementedError('Unsupported search operator')
+        if operator == '!=':
+            value = not value
+        return expression.TRUE_DOMAIN if value and self.user_is_admin \
```

Comment:
does this really work? won't this recursively check for `self.user_is_admin` and make an infinite loop

Reply:
The `search` method is actually not necessary. By removing that, I noticed that the field no longer appears in the 'Custom Filter' section of the list view. This is actually what I wanted :smile: